From 9ddd6026e0cdb8fdb43f9d576dbaea1339145a68 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 11 Apr 2024 01:00:13 +0800
Subject: [PATCH] fix(CMakeLists.txt): fix scalable SVG icon installation path

Should be /usr/share/icons/hicolor/scalable/apps rather than simply
/usr/share/icons.

Ref: https://src.fedoraproject.org/rpms/nvtop/blob/3f30e8a9e38ed42beead63c3bbc8a240e87140e0/f/nvtop-fix-icon-path.diff
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b0e5679..550edfe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -121,7 +121,7 @@ install(FILES
   RENAME nvtop.1)
 install(FILES
   "${CMAKE_CURRENT_SOURCE_DIR}/desktop/nvtop.svg"
-  DESTINATION share/icons
+  DESTINATION share/icons/hicolor/scalable/apps
   PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
 install(FILES
   "${CMAKE_CURRENT_SOURCE_DIR}/desktop/nvtop.desktop"
-- 
2.44.0

